pegasus/sites.v3/code.org/public/online-pl.redirect

```diff
@@ -1 +1 @@
-https://code.org/educate/professional-development-online
+/professional-learning/self-paced
```

Comment:
/educate/professional-development-online [also redirects](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/educate/professional-development-online.redirect) to /professional-learning/self-paced, so I updated to avoid the double redirect